SIVA/Core/Files/Objects/Server.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+
+namespace SIVA.Core.Files.Objects
+{
+    public class Server
```

Comment:
Split configuration into multiple configuration objects to allow for a smaller file where modules are disabled.

Reply:
compatibility with V1